ucsb_ds_capstone_projects_2021/projects/appfolio/update2.md

```diff
@@ -1,38 +1,52 @@
+ ```
+Before tokenization:  This is the LayoutLM tokenizer!
+After tokenization:  [101, 1188, 1110, 1103, 22002, 3554, 22074, 22559, 17260, 106, 102]
+```
 
-* As an alternative approach, we started our implementation of using object detection to classify key data within the invoice. Object detection is an expansion of image classification, in which rather than simply classifying an image, the detector outputs the localized bounding box coordinates as well as the predicted class. Our team decided to go with Detectron2, Facebook’s open source object detection library. The main reasoning behind this decision was based on Detectron2’s capability of achieving high confidence with a relatively simple structure, as well as having ample collections of pretrained models for various object detection algorithms.
 
-* In regards to data access, we have finally gotten access to the Appfolio dataset, which is composed of:
-  * images of the invoices
-  * csv files composed of bounding box annotations for each word in the invoices
-  * the metadata of each invoice upload, which includes a Total amount value that we can use as validation for our models that extract the total amount from an invoice
+![FUNSDpred](https://github.com/DSCapstone2021/ucsb-ds-capstone-2021.github.io/blob/main/ucsb_ds_capstone_projects_2021/projects/appfolio/FUNSDpred.png?raw=true)
```

Comment:
For the image `FUNSDpred.png` in the same directory as the markdown file, you can use `![some-text](FUNSDpred.png)` 
Refer to https://jupyterbook.org/content/figures.html for more info